src/core/thread/thread_netif.hpp

```diff
@@ -205,6 +206,9 @@ class ThreadNetif : public Ip6::Netif
 #if OPENTHREAD_CONFIG_SRP_CLIENT_ENABLE
     Srp::Client mSrpClient;
 #endif
+#if OPENTHREAD_CONFIG_DNSSD_SERVER_ENABLE
+    Dns::ServiceDiscovery::Server mDnsServer;
```

Comment:
Do we want to name it as `mDnsServer` or `mDnsSdServer (/mDnssdServer)`?

Reply:
I would prefer `mDnssdServer` . Thoughts?

Reply:
Renamed to mDnssdServer